kubernetes/helm_charts/datapipeline_jobs/values.j2

```diff
@@ -841,16 +876,23 @@ live-node-publisher:
     enableDIALContextUpdate = "Yes"
 
     cloud_storage_type="{{ cloud_store }}"
-    azure_storage_key="{{ sunbird_public_storage_account_name }}"
-    azure_storage_secret="{{ sunbird_public_storage_account_key }}"
-    azure_storage_container="{{ azure_public_container }}"
+    cloud_storage_key="{{ cloud_storage_key }}"
```

Comment:
@anilguptatarento Check the env name
https://docs.google.com/spreadsheets/d/1PSaQfPXvv_rkCAjnMR-1aJHC15O0jGQ_pZ1ER4l_2BM/edit#gid=0

It should be "cloud_public_storage_secret" not "cloud_storage_key"

Reply:
Check this once across our repos